lib/cdc/mongo/debezium.go

```diff
@@ -162,6 +162,10 @@ func (s *SchemaEventPayload) GetData(pkMap map[string]any, tc *kafkalib.TopicCon
 		}
 
 		retMap[constants.DeleteColumnMarker] = true
+		// For now, assume we only want to set the deleted column and leave other values alone.
+		// If previous values for the other columns are in memory (not flushed yet), [TableData.InsertRow] will handle
+		// filling them in and setting this to false.
+		retMap[constants.OnlySetDeleteColumnMarker] = true
```

Comment:
This is where we first set this marker; we set it to true if it's a delete event, false otherwise

Reply:
Add a test?